script.js

```diff
@@ -227,20 +213,23 @@ const cellElem = d.querySelectorAll('[data-cell]');
 // 1. Shuffle all cards in database
 shuffleCards(Cards);
 
-// 2. Distribute to players decks
-createDecks(Cards, playerDeck, opponentDeck);
+// 2. Distribute to decks
+createDecks(Cards, blueDeck, redDeck);
 
 // 3. Deal Deck
-deal(playerDeck, playerHand);
-deal(opponentDeck, opponentHand);
+deal(blueDeck, blueHand);
+deal(redDeck, redHand);
+
+// 4. Random First Turn
+randomFirstTurn();
```

Comment:
Agreed! It would give a better approximation of what further iterations of this project will ultimately look like. Also with the implementation of the start/new game function that is upcoming, it would make sense to not have to call each function individually again.

Reply:
Haha yes, for sure!